.github/workflows/ci.yml

```
@@ -0,0 +1,15 @@
+---
+name: CI
+on: [push, pull_request]
```

```suggestion
on:
  push:
    branches: master
  pull_request: []
```
to avoid duplicate results (like we have here)